codecov.yml

```diff
@@ -0,0 +1,38 @@
+codecov:
+  require_ci_to_pass: yes
+
+coverage:
+  status:
+    project:
+      default:
+        # basic
+        target: auto
+        threshold: 5%
+        base: auto
+        flags:
+          - unit
+        # advanced
+        branches:
+          - master
```

Comment:
All branches should be able to trigger the Codecov action. 
As other CI testing is triggered only for master commit and pull request actions, Codecov action will be triggered in the same way.